docs/triggers_rules.md

```diff
@@ -82,6 +82,7 @@ As an example, create a trigger to send user location updates, and manually fire
 
    Any events that you fire to the statusUpdate trigger currently don't do anything. To be useful, the trigger needs a rule that associates it with an action.
 
+Triggers can't be created inside a package, they must be created directly under a namespace.
```

Comment:
I going to change to "Triggers may not be created inside a package, they must be created directly under a namespace."

>Triggers are only allowed in the default package. (Do we defined default package)?
What are you referring here by "default package"?